### DISCUSSION TITLE
Gizmo for RigidBody center of mass

### DISCUSSION THREAD

**Discussion by RBerezkin** [godotengine/godot-proposals#6372] [Editor]
In Godot 4, it finally became possible to change the position of the center of mass for a rigid body. It would be nice to make this position visible in the editor window so that the user can see where it is relative to the body. Now I have to create a helper node, move it and then copy the coordinates.

**Comment by Calinou** [comment]
This is already being tracked in https://github.com/godotengine/godot-proposals/issues/2579.
